sources/packages/web/src/services/http/dto/Restriction.dto.ts

```diff
@@ -45,6 +45,16 @@ export interface AssignRestrictionAPIInDTO extends ResolveRestrictionAPIInDTO {
   restrictionId: number;
 }
 
+export interface AssignRestrictionCategories {
```

Comment:
Are these interfaces used anywhere?
Besides that, they look like interfaces to populate the dropdowns(or similar components), if so we can have a generic interface.